### PR TITLE
[redwood] Use new esbuild script

### PR DESCRIPTION
**Pull request by cb1kenobi**

### CODE REVIEW

**Review by styfle — CHANGES_REQUESTED** [review]
Same comments as the others

File: .changeset/few-moons-unite.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (TooTallNate) [inline .changeset/few-moons-unite.md:3]: Patch

File: packages/redwood/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "directory": "packages/redwood"
   },
   "scripts": {
-    "build": "node build.js",
+    "build": "node ../../utils/build.mjs",
```
Comment (TooTallNate) [inline packages/redwood/package.json]:
```suggestion
    "build": "node ../../utils/build-builder.mjs",
```

File: .changeset/few-moons-unite.md (hunk 1 above)
Comment (cb1kenobi) [reply .changeset/few-moons-unite.md:3]: Fixed

File: packages/redwood/package.json (hunk 2 above)
Comment (cb1kenobi) [reply packages/redwood/package.json]: Fixed

**Review by styfle — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**